notebooks/2021_tests/gather_metrics.sql

```diff
@@ -33,6 +33,17 @@ WITH relevant_fields AS (
      AND '20211125'
     ),
 
+-- get visitors who visited at least one page with node2vec related links that day
+filtered_visitors AS (
+    SELECT 
+        DISTINCT date, fullVisitorId
+    FROM relevant_fields 
+    WHERE content_id NOT IN (
+    SELECT content_id 
+    -- list of content IDs with manually curated related links, generated using notebooks/2021_tests/get pages with manually curated links.ipynb
+    FROM `govuk-bigquery-analytics.datascience.manual_related_links_pages`)
```

Comment:
```suggestion
    FROM `govuk-bigquery-analytics.datascience.manual_related_links_pages`)
     AND doc_type NOT IN (
        "html_publication",
        "homepage",
        "organisation",
        "finder",
        "mainstream_browse_page",
        "coronavirus_landing_page",
        "travel_advice_index	",
        "service_sign_in",
        "step_by_step_nav",
        "topic"
    )
```

I got too into this and looked at GA to find document types that are popular but don't see any related link click events, probably because the layout of the pages doesn't display any related links, and added a bit the the query above to select the document type custom dimension

```
(SELECT
      value
    FROM
      hits.customDimensions
    WHERE
      index=2) AS doc_type